.github/workflows/ci.yml

```diff
@@ -6,6 +6,9 @@ on:
   pull_request:
     branches: [main]
 
+env:
+  GRAPH_API_KEY: 0e7dd738749485cb3dceaeaa33ce3eb8
```

Comment:
Is this expected to be exposed? I guess there is no other way around because of how GitHub secrets work in workflows...

Reply:
Probably for CI purposes we should be using Alchemy URLs if that's possible to have across all networks.

Reply:
Yep, exactly. I don't see any other way. I made sure that this API key has sensible limits and restrictions so any potential damage would be extremely small. I'd say let's give this a go for now.

The problem with Alchemy is that we downgraded to the free plan and now only have 2 deployments available. We decided to have 2 mainnets deployed to be used for local deployments and Vercel previews. 
At any rate, if we re-enable token price tests from DEX Subgraphs we'll have to use The Graph anyway because this is where they're hosted.